client/src/ContactMe.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react'
+import FooterForm from './FooterForm'
+
+// contact me form to submit an email
+
+export class ContactMe extends Component {
+    constructor(props){  
+        super(props);  
+
+        this.state= {
+         name: '',
+         email: '',
+         message: ''
+    
+        }  
+        
+    }
+    
+
+	onNameChange = (event) => {
+		this.setState({ name: event.target.value });
+	};
+
+	onEmailChange = (event) => {
+		this.setState({ email: event.target.value });
+	};
+
+	onMessageChange = (event) => {
+		this.setState({ message: event.target.value });
+  };
+  
+  handleSubmit=(e)=> {
+    e.preventDefault();
+
+    fetch('http://localhost:3000/contact',{
+        method: "POST",
+        body: JSON.stringify(this.state),
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        },
```

Comment:
You are going to have to adjust your fetch / nodemailer process. When trying to send mail from your heroku app, it throws a CORS error. I believe this is because you are trying to fetch from localhost but your site is hosted on heroku, so it can't actually fetch from localhost. I'm not super familiar with nodemailer, but I know there are docs for nodemailer which you could use to try to fix the issue. One step would be to use ```fetch('/contact'``` which I believe would fetch from the current path/contact. It may be a simple fix like that, or there may be more to it, but look into this!

A couple articles on setting up a form and nodemailer to send emails: [Here](https://tylerkrys.ca/blog/adding-nodemailer-email-contact-form-node-express-app) and [here](https://medium.com/@binhchung48/create-a-contact-form-with-nodemailer-react-js-and-express-js-7757d41e2448).